rfd/0067-desktop-access-file-system-sharing.md

```diff
@@ -594,3 +594,204 @@ The UX for this feature is discussed in [RFD 0058](https://github.com/gravitatio
 ## Security
 
 Security will be discussed in [a future RFD](https://github.com/gravitational/teleport/issues/13406).
+
+## Audit Events
+
+Audit events would ideally be focused exclusively on directory sharing events that are germaine to system security such as:
+
+- A file was transferred from the remote Windows machine onto the local machine (into the shared directory)
+- A file was transferred from the local machine (from the shared directory) onto the remote Windows machine
+- etc
+
+However due to our limited visibility into the Windows side of the equation it is not possible for us to determine such events with precision. For example, if a user transfers a file
+from the Windows box into the shared directory, we see that as a `Shared Directory Create` sequence followed by a `Shared Directory Write` sequence. But then that same sequence could
+just have been a user creating a brand new file within the shared directory and then writing to it (which is not of particular note from a security perspective). Given this limitation,
+we will instead simply log all events that could conceivably indicate security-relevant information, and document the expected sequence of events for such common scenarios such as those
+described above for our users to reference. `Shared Directory Announce/Acknowledge` is also included in the audit event log to make the sequence of events more easily comprehensible.
+
+**TDP Shared Directory Messages Logged**
```

Comment:
Let's make this less about messages and more about events. For example, list the proposed event name and when it occurs.

- `windows.desktop.directory.share`: indicates that the user has shared a directory with the remote desktop
- `windows.desktop.directory.{read|write}`: etc...

Reply:
I went with `"desktop.directoy.*"` based on other events here
https://github.com/gravitational/teleport/blob/f87519333620dd5105b376ff2edfc8b71279080d/lib/events/api.go#L545-L552

Though I noticed that our start and end events are of the `"windows.desktop.*"` form. Perhaps we should modify one or the other to stick to a single pattern?

https://github.com/gravitational/teleport/blob/f87519333620dd5105b376ff2edfc8b71279080d/lib/events/api.go#L528-L533